modules/m_who.c

```diff
@@ -565,6 +568,21 @@ do_who(struct Client *source_p, struct Client *target_p, struct membership *mspt
 				q = "0";
 			append_format(str, sizeof str, &pos, " %s", q);
 		}
+		if (fmt->fields & FIELD_GATEWAY)
+		{
+			/* use same the logic as account */
```

Comment:
Given the reasoning for account to do this [1], I don't think it makes sense to copy that logic for gateway.

[1]: https://github.com/solanum-ircd/solanum/blob/0ca18d072a7d1dfcfbdff6dcf4f2788f52935a03/modules/m_services.c#L348-L350